### PR TITLE
Moving parts of the Shape Registry into a common file

### PR DESCRIPTION
**Pull request by Gamrix**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #67575
* #67500
* #66973
* __->__ #66948

Differential Revision: [D32497550](https://our.internmc.facebook.com/intern/diff/D32497550)

### CODE REVIEW

**Review by Krovatkin — APPROVED** [review]
:shipit:

File: torch/csrc/jit/passes/utils/op_registry.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      "aten::replication_pad3d(Tensor self, int[] padding) -> Tensor",
+      "aten::upsample_bilinear2d(Tensor self, int[] output_size, bool align_corners, float? scales_h, float? scales_w) -> Tensor",
+      "aten::upsample_linear1d(Tensor self, int[] output_size, bool align_corners, float? scales) -> Tensor",
+      "aten::upsample_nearest1d(Tensor self, int[] output_size, float? scales) -> Tensor",
+      "aten::upsample_nearest2d(Tensor self, int[] output_size, float? scales_h, float? scales_w) -> Tensor",
+      "aten::upsample_nearest3d(Tensor self, int[] output_size, float? scales_d, float? scales_h, float? scales_w) -> Tensor",
+      "aten::upsample_trilinear3d(Tensor self, int[] output_size, bool align_corners, float? scales_d, float? scales_h, float? scales_w) -> Tensor",
+      "aten::prelu(Tensor self, Tensor weight) -> Tensor",
+  });
+  return ops;
```
Comment (Krovatkin) [inline torch/csrc/jit/passes/utils/op_registry.cpp:56]: I wonder why they didn't use a plain reference instead, but it isn't something you changed or introduced.

**Review by Krovatkin — APPROVED** [review]
:shipit:

**Comment by Gamrix** [comment]
@gamrix has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32175960).

**Comment by Gamrix** [comment]
@gamrix has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32175960).

**Comment by Gamrix** [comment]
@gamrix has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32497550).

**Pull request closed without merging**